app/utils/ApiClient.ts

```diff
@@ -5,7 +5,7 @@ import {createLogger} from "./Logger";
 const logger = createLogger("ApiClient.ts")
 
 const apiClient = axios.create({
-  baseURL: "https://demo.openboxes.com/openboxes/api",
+  baseURL: "https://demo2.openboxes.com/openboxes/api",
```

Comment:
In the future we'll obviously want this to be a configuration property.

Reply:
Sure. Will setup environment config to handle local, development and production configuration.